### DISCUSSION TITLE
Upgraded to python 3.9 and getting warnings

### DISCUSSION THREAD

**Discussion by jamhot1** [kernc/backtesting.py#217] [General]
Getting the below warning now that I have upgraded to 3.9. Unsure if this is related to Pandas/numpy or the library.

Any of the smart cookies among us recognise what this means?

python3.9/site-packages/backtesting/backtesting.py:1042: FutureWarning: Index.is_all_dates is deprecated, will be removed in a future version.  check index.inferred_type instead
  if (not data.index.is_all_dates

The Algo still runs fine but have a feeling things may break soon.

**Comment by kernc** [comment]
Fixed already in https://github.com/kernc/backtesting.py/commit/c2c334385a2598dd7e775b928f7eec568e5aba97, will be in next release.

This, as an issue/todo item, would belong on the issue tracker.
